src/pageEditor/panes/EditorPane.test.tsx

```diff
@@ -262,7 +262,7 @@ async function addABlock(addButton: Element, blockName: string) {
 
 describe("renders", () => {
   beforeEach(() => {
-    // :barf: these Jest snapshot contains sequence UUIDs
+    // :barf: these Jest snapshots contains sequence UUIDs
```

Comment:
Really tiny, sorry, just noticed because you changed to plural on the other word here...
```suggestion
    // :barf: these Jest snapshots contain sequence UUIDs
```